arithmetic_analysis/newton_backward_interpolation.py

```diff
@@ -0,0 +1,57 @@
+# https://www.geeksforgeeks.org/newton-forward-backward-interpolation/
+from __future__ import annotations
```

Comment:
Why are you importing this? I see no typehints that aren't supported in later versions

Reply:
Sure, I am removing it